Sources/Scout/Implementations/XML/PathExplorerXML+Get.swift

```diff
@@ -11,13 +11,13 @@ extension PathExplorerXML {
     // MARK: - Array
 
     /// - parameter negativeIndexEnabled: If set to `true`, it is possible to get the last element of an array with the index `-1`
-    /// - parameter ignoreArraySlicing: If `true`, the fact that self is part of a slicing will be ignore to simple get the index
-    func get(at index: Int, negativeIndexEnabled: Bool = false) throws -> Self {
+    /// - parameter detailedName: If`true`, when using a dictionary filter, the keys names will be changed to reflect the filtering
```

Comment:
Missing a space after the `If`.

Reply:
Fixed in latest commit 739ca313022c667ae53fc96f79a81c50d6a8264f